.github/workflows/release.yml

```diff
@@ -155,18 +168,25 @@ jobs:
           echo "GitHub event: $GITHUB_EVENT"
 
       - name: Checkout repo
-        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
         with:
           ref: master
 
-      - name: Setup project name
+      - name: Setup project name and release tag
         id: setup
         run: |
           PROJECT_NAME=$(echo "${{ matrix.project_name }}" | awk '{print tolower($0)}')
           echo "Matrix name: ${{ matrix.project_name }}"
           echo "PROJECT_NAME: $PROJECT_NAME"
           echo "project_name=$PROJECT_NAME" >> $GITHUB_OUTPUT
 
+          if [ -z "${{ matrix.release_tag }}" ]; then
+            # Use core release tag by default.
+            echo "RELEASE_TAG=$_CORE_RELEASE_TAG" >> $GITHUB_OUTPUT
+          else
+            echo "RELEASE_TAG=${{ matrix.release_tag }}" >> $GITHUB_OUTPUT
+          fi
+
```

Comment:
Would it makes sense to use the `matrix.release_tag` pattern for the server projects as well and deprecate this branching logic?

Reply:
Yeah, that makes sense